### ISSUE TITLE
pseudo seen and pseudo unseen domains

### ISSUE THREAD

**Issue by xzhangeb** [hytseng0509/CrossDomainFewShot#8]
Hi,
I found in your code that every epoch you split the ps and pu domains by random.sample(base_set, k=2), so you only use one domain as the ps domain and one another domain as the pu domain during your experiment?

**Comment by hytseng0509** [comment]
We use one domain as the ps domain and one another domain as the pu domain for **the same training epoch**.
Ideally, we should sample different sets for ps and pu domains for each training iteration. However, it is not very efficient as we need to re-configure the data loader for each training iteration.
To address that issue, we perform the domain sampling for each training epoch.

**Comment by yxgnahz** [comment]
Thanks for your response. I re-implemented your model in my work, and I observed that even if I load the pre-trained model, the validation accuracy will drop dramatically once starting back propagation with FT layer enabled (I tried to train on miniImagenet and test on CUB). Did you observe this during your training process?

**Comment by hytseng0509** [comment]
No, I don't observe that problem. As long as the values of the transformation parameters in the FT layers are in a reasonable interval, the FT layers should help generalize the metric-based model.
